spec/graphql/directive_spec.rb

```diff
@@ -8,6 +8,7 @@
         # plain fields:
         skipFlavor: flavor @skip(if: true)
         dontSkipFlavor: flavor @skip(if: false)
+        dontSkipDontIncludeFlavor: flavor @skip(if: false), @include(if: false)
```

Comment:
Good idea, added `skipAndIncludeFlavor` which should be skipped if skip has precedence.